### ISSUE TITLE
Support of sexagesimal variable type

### ISSUE THREAD

**Issue by blourdel** [Juniper/open-nti#107]
A 'show command' may report a counter value in the form of a sexagesimal value.
E.g : mm:ss , hh:mm:ss 

This should be converted to integer : hh\*3600+60\*mm+ss  in the database, so graphing can be performed.

at present in the open-ntyi.py file

def eval_variable_value(value,**kwargs):

only process integer or string types.

**Comment by 3fr61n** [comment]
Hi @blourdel 

Could you please tell me which command are you trying to collect that contains timestamp?  I'll try to check what options do we have.

Regards

**Comment by blourdel** [comment]
Hi,

I am issuing :
regress@hongkong> show system processes extensive
........
  PID USERNAME  THR PRI NICE   SIZE    RES STATE   C   TIME    WCPU COMMAND
   10 root        4 155 ki31     0K    64K RUN     3 431.0H 390.19% idle
 9738 root        1  34    0   725M 12044K select  0   0:12   5.18% python
 5280 root        4  20    0  5194M  4255M kqread  1 145:33   0.29% rpd

I am after the TIME column :  0:12 145:33 
a potential value is 431.0H as well but not sure if we can deal with it.

**Comment by 3fr61n** [comment]
Yes you're right, that field changes it format across time,  at the beginning of this thread I though you were asking other timestamp fields in other junos commands, in those cases I use the xml attribute junos:seconds as a feasible workaround, but sadly is not your case

` <date-time junos:seconds="1481112868">2016-12-07 13:14:28 CET</date-time>`

**Comment by dgarros** [comment]
FYI, I'm working on a big refresh of the netconf collector, I think this enhancement could be done as part of this effort. 
Hopefully I'll have more to share in 1 week
